regolith/helpers/u_milestonehelper.py

```diff
@@ -23,9 +23,15 @@
 def subparser(subpi):
     date_kwargs = {}
     notes_kwargs = {}
+    status_kwargs = {}
+    type_kwargs = {}
     if isinstance(subpi, GooeyParser):
         date_kwargs['widget'] = 'DateChooser'
         notes_kwargs['widget'] = 'Textarea'
+        status_kwargs['widget'] = 'Dropdown'
+        status_kwargs['choices'] = ALLOWED_STATI
```

Comment:
I think choices are allowed (and desirable) in `ArgumentParser()` so maybe we should move these out of the gooey_kwargs

Reply:
Some helper tests were failing when I had 'choices' in the subparser, so I put it in the gooey_kwargs. I can try to get the tests to work with 'choices' in the subparser.

Reply:
yes, better.  Always try to leave code better than you found it.     Check the tests are testing the right thing.....and are they covering all the possibilities that can be tested?  For example, if you give a wrong choice (a) what do you want the program to do and (b) what is it doing.  The test should capture (a) and refactoring the code so that (b) = (a).

Reply:
always start with (a)